static/css/blog.css

```diff
@@ -19,15 +19,15 @@ html body .region-inner {
 
 a:link {
   text-decoration:none;
```

Comment:
@Abhik1998 Thank you for the fix again!

This change will affect all the links which are supposed to be in blue. Refer the below screenshot/ verify on the Netlify deployment. 

<img width="1229" alt="screen shot 1940-12-07 at 1 47 19 pm" src="https://user-images.githubusercontent.com/6227784/53397508-4b8b2580-39cd-11e9-9c36-0de0d9e2d047.png">